src/compojure_throttle/core.clj

```diff
@@ -25,42 +25,60 @@
   (Integer. (or (env key)
                 (defaults key))))
 
+(defn make-throttle-cache
+  "Initialize a throttling cache with a given TTL and tokens.
+
+   Pass a map containing keys :ttl and :tokens.
+
+   This maintains all of the state for figuring out which request to throttle.
+   (Yeah, it's just a TTL cache, but don't rely on that.)"
+  [{:keys [ttl tokens]}]
+  {:cache          (atom (cache/ttl-cache-factory {} :ttl ttl))
+   :ttl-setting    ttl
+   :tokens-setting tokens})
```

Comment:
why don't you use just `ttl` ?  `ttl-setting` name seems like `map` value. also `tokens-settings` , how about `token-count` ? , I couldn't think tokens-setting is `number` value

Reply:
https://github.com/kirasystems/compojure-throttle/pull/4/commits/f16141d3fbf1724d78b4d806bb7641353bf25680